plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeAnalyzeProperties.java

```diff
@@ -39,6 +40,7 @@
 {
     public static final String FILES_MODIFIED_AFTER = "files_modified_after";
     public static final String COLUMNS_PROPERTY = "columns";
+    public static final String FORCE_RECALCULATE_STATISTICS = "force_recalculate_statistics";
```

Comment:
Pls create a follow-up docs task if you don't intend now to document this new option.

Reply:
i forgot about docs, i will add them here